addons/website/static/src/snippets/s_table_of_content/options.js

```diff
@@ -56,7 +56,15 @@ options.registry.TableOfContent = options.Class.extend({
             $el.attr('id', id);
             $el[0].dataset.anchor = 'true';
         });
-        $nav.find('a:first').addClass('active');
+
+        const targetNodeEl = this.$target[0].querySelector('.s_table_of_content_main');
+        if (targetNodeEl && targetNodeEl.children.length === 0) {
```

Comment:
>Why do you need to test if targetNodeEl exists ?

for v15 (removing content with backspace) it will lead to removing the `targetNodeEl`, so in the end, we do not have `targetNodeEl` and that leads us to TB `targetNodeEl is null` if we are not testing for `targetNodeEl`